apps/src/bin/tquic_client.rs

```diff
@@ -801,17 +870,74 @@ impl RequestSender {
             self.option.max_requests_per_conn
         );
 
+        // update requests to be sent
+        self.update_new_reqs();
+
+        // TODO: 防止读到尾导致数组index溢出
```

Comment:
Change if to English of the comments. Or if it has been done, remove the TODO comments.